frontend/src/main/scala/bloop/bsp/BloopBspServices.scala

```diff
@@ -863,6 +864,13 @@ final class BloopBspServices(
   private def toBuildTargetId(project: Project): bsp.BuildTargetIdentifier =
     bsp.BuildTargetIdentifier(project.bspUri)
 
+  private def toJvmBuildTarget(project: Project): bsp.JvmBuildTarget = {
+    val javaHome = Option(System.getProperty("java.home")).map(s => bsp.Uri(Paths.get(s).toUri))
```

Comment:
I just learned from an offline discussion with @jvican that Bloop supports configuring custom Java home per-project. See `.platform` here https://github.com/scalacenter/bloop/blob/906110dc384dbebb8d7d1b2f62bd226231a4f3bf/frontend/src/main/scala/bloop/data/Project.scala#L52

We should use `JdkConfig.javaHome` instead of the `java.home` system property.